index.js

```diff
@@ -806,13 +776,16 @@
   //. In general terms, `compose` performs right-to-left composition of two
   //. unary functions.
```

Comment:
It's not really related, that's true, but an issue entitled "improve all function descriptions" is too daunting to consider. We can at least make small improvements as they occur to us. :)